pontoon/base/models.py

```diff
@@ -1796,6 +1796,20 @@ def aggregate_stats(self):
         ).distinct().aggregate_stats(self)
 
 
+def repository_url_validator(url):
+    # Regular URLs
+    validator = URLValidator(["http", "https", "ftp", "ftps", "ssh", "svn+ssh"])
+
+    # Git SCP-like URL
+    pattern = r"(?P<user>git@([\w\.@]+)(/|:))(?P<server>[\w,\-,\_]+)/(?P<project>[\w,\-,\_]+)(.git){0,1}((/){0,1})"
```

Comment:
```suggestion
    pattern = r"git@[\w\.@]+[/:][\w-]+/[\w-]+(.git)?/?"
```

Simpler this way, and the previously named "server" and "project" groups shouldn't include `,`.

Reply:
Good catch!